backbone/src/main/java/org/researchstack/backbone/model/ProfileInfoOption.java

```diff
@@ -0,0 +1,101 @@
+package org.researchstack.backbone.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by TheMDP on 1/4/17.
+ *
+ * Used by several differnt Step types to designate re-usable QuestionStep types
+ * that collect user profile info
+ */
+
+public enum  ProfileInfoOption {
+    @SerializedName("email")
+    EMAIL("email"),
+    @SerializedName("password")
+    PASSWORD("password"),
+    @SerializedName("externalID")
+    EXTERNAL_ID("externalID"),
+    @SerializedName("name")
+    NAME("name"),
+    @SerializedName("birthdate")
+    BIRTHDATE("birthdate"),
+    @SerializedName("gender")
+    GENDER("gender"),
+    @SerializedName("bloodType")
+    BLOOD_TYPE("bloodType"),
+    @SerializedName("fitzpatrickSkinType")
+    FITZPATRICK_SKIN_TYPE("fitzpatrickSkinType"),
+    @SerializedName("wheelchairUse")
+    WHEEL_CHAIR_USE("wheelchairUse"),
+    @SerializedName("height")
+    HEIGHT("height"),
+    @SerializedName("weight")
+    WEIGHT("weight"),
+    @SerializedName("wakeTime")
+    WAKE_TIME("wakeTime"),
+    @SerializedName("sleepTime")
+    SLEEP_TIME("sleepTime");
+
+    String identifier;
+    public String getIdentifier() { return identifier; }
+
+    /** Only useful for when Enum == PASSWORD */
+    boolean addConfirmPassword = false;
```

Comment:
Good find on this, definitely shouldn't be in the enum.  I was able to move it to be a parameter within SurveyFactory.